src/java/org/apache/cassandra/tools/nodetool/stats/CompactionHistoryPrinter.java

```diff
@@ -65,14 +67,15 @@ public void print(CompactionHistoryHolder data, PrintStream out)
             for (Object chr : compactionHistories)
             {
                 Map value = chr instanceof Map<?, ?> ? (Map)chr : Collections.emptyMap();
-                String[] obj = new String[7];
+                String[] obj = new String[8];
                 obj[0] = (String)value.get("id");
                 obj[1] = (String)value.get("keyspace_name");
                 obj[2] = (String)value.get("columnfamily_name");
                 obj[3] = (String)value.get("compacted_at");
                 obj[4] = value.get("bytes_in").toString();
                 obj[5] = value.get("bytes_out").toString();
                 obj[6] = (String)value.get("rows_merged");
+                obj[7] = value.get("compaction_type") == null ?  UNKNOW_TYPE : (String)value.get("compaction_type");
```

Comment:
this should be `obj[7] = value.get("compaction_type");` because it already contains type or `UNKNOWN`.

Reply:
+1